### PR TITLE
Add secondary nav and new /internet-of-things/core page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Add secondary nav for IoT section
- Add new core page at `/internet-of-things/core` - refactored this [branch](https://github.com/zprood/cn.ubuntu.com/blob/master/templates/core/index.html) as the page wasn't responsive and the headings weren't ordered correctly. Also extracted the styles from the html file and injected them into scss files.

## QA

- Visit page at: https://cn-ubuntu-com-496.demos.haus/internet-of-things
- Check secondary nav for IoT - described [here](https://github.com/canonical-web-and-design/web-squad/issues/3890)
- `/internet-of-things/core page` content should match ubuntu.com/core - copy [here](https://github.com/zprood/cn.ubuntu.com/blob/master/templates/core/index.html)

## Issue / Card

[#3890](https://github.com/canonical-web-and-design/web-squad/issues/3890)

## Screenshots

image

### CODE REVIEW

File: static/sass/_pattern_core.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
+@mixin ubuntu-p-core {
+  @media only screen and (min-width: 461px) {
+    .p-banner-core-hero {
```
Comment (anthonydillon) [inline static/sass/_pattern_core.scss:3]: Doesn't look like this is used

File: templates/internet-of-things/core.html [hunk 2]
```diff
@@ -0,0 +1,1171 @@
...
+        </p>
+      </div>
+      <div class="col-5 u-align--center u-embedded-media">
+        <iframe title="UC20 video" class="u-embedded-media__element" src="//player.bilibili.com/player.html?aid=501588719&bvid=BV1YN411R7pY&cid=295296600&page=1" frameborder="0" framespacing="0" allowfullscreen=""></iframe>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light u-no-padding--top">
+    <div class="row u-vertically-center u-align--center">
```
Comment (anthonydillon) [inline templates/internet-of-things/core.html:28]: The `u-align--center` here is stoping the row doing its thing. Can you add a `col-12` div inside with the align class

**Comment by zprood** [comment]
@bethcollins92 Thank you, I notice that there is a problem on the /core page with the second and third section if resizing the page towards the horizontal direction. It's working with 1080p resolution and lower but broken over 1080P, like 2K, 3K, 4K. 
image

File: templates/internet-of-things/core.html [hunk 3]
```diff
@@ -0,0 +1,1171 @@
...
+  </section>
+  
+  <!-- start generated content -->
+  
+  <section class="p-strip">
+    <div class="u-fixed-width u-sv3">
+      <h2>关于智能用例</h2>
+      <p class="p-heading--4">获取Ubuntu Core、获取snap应用、获取经济效益</p>
+    </div>
+    <div class="row u-equal-height u-align--center u-vertically-center u-hide--small">
```
Comment (anthonydillon) [inline templates/internet-of-things/core.html:221]: Again we should align at the column level.

File: templates/internet-of-things/core.html [hunk 4]
```diff
@@ -0,0 +1,1171 @@
...
+          width="144",
+          height="126",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+        </div>
+      </div>
+    </div>
+    <div class="row u-align--center">
```
Comment (anthonydillon) [inline templates/internet-of-things/core.html:262]: Here too

File: templates/internet-of-things/core.html [hunk 5]
```diff
@@ -0,0 +1,1171 @@
...
+      </div>
+      <div class="col-4">
+        <p class="p-heading--4 u-sv2"><a href="https://ubuntu.com/embedded">嵌入式&nbsp;&rsaquo;</a></p>
+      </div>
+      <div class="col-4">
+        <p class="p-heading--4 u-sv2"><a href="https://ubuntu.com/internet-of-things/networking">IoT&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  
+    <div class="row u-equal-height u-align--center u-vertically-center u-hide--small">
```
Comment (anthonydillon) [inline templates/internet-of-things/core.html:274]: And here

File: templates/internet-of-things/core.html [hunk 6]
```diff
@@ -0,0 +1,1171 @@
...
+            width="120",
+            height="121",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+    <div class="row u-align--center">
```
Comment (anthonydillon) [inline templates/internet-of-things/core.html:315]: And here

File: templates/internet-of-things/core.html [hunk 7]
```diff
@@ -0,0 +1,1171 @@
...
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="col-6">
+        <h3 class="p-heading--4">安全启动</h3>
+        <h4 class="p-heading--5">具备硬件信任的系统完整性</h4>
+        <p>确保您所有的设备都运行您选择的系统软件，并抵御X86和ARM上的底层启动攻击。</p>
+        <p>
+          <a href="https://ubuntu.com/core/features/secure-boot">了解更多关于安全启动的内容&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/internet-of-things/core.html:484]: This should be an external link

File: templates/internet-of-things/core.html [hunk 8]
```diff
@@ -0,0 +1,1171 @@
...
+      <hr class="p-separator" />
+    </div>
+  
+    <div class="row u-vertically-center">
+      <div class="col-6">
+        <h3 class="p-heading--4">全盘加密</h3>
+        <h4 class="p-heading--5">保护敏感用户数据</h4>
+        <p>使用TPM或TEE中的硬件密钥将数据锁定到设备，并仅可被您系统软件解锁，以阻止对敏感信息的物理攻击。</p>
+        <p>
+          <a href="https://ubuntu.com/core/features/full-disk-encryption">了解更多关于全盘加密的内容&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/internet-of-things/core.html:499]: Same here

File: templates/internet-of-things/core.html [hunk 9]
```diff
@@ -0,0 +1,1171 @@
...
+        </div>
+        <h3 class='p-heading--four'><a href='https://cn.ubuntu.com/engage/ubuntu-tech-live-episode-4' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'core - center', 'eventLabel' : 'Webinar - rule4 security Ubuntu Core', 'eventValue' : '1' });'>Ubuntu Core的网络安全分析</a></h3>
+        <!-- rtp section center end -->
+      </div>
+    </div>
+  </section>
+  
+  <section class='p-strip--square-suru is-slanted--top-right'>
+    <div class="row">
+      <h2>通过SMART START快速进入市场</h2>
```
Comment (anthonydillon) [inline templates/internet-of-things/core.html:1144]: Can you put these in a column please

**Comment by bethcollins92** [comment]
@zprood ah I'll take a look, thanks!

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looks good with a few comment in the code

**Comment by bethcollins92** [comment]
Thanks @anthonydillon, applied those fixes 👍🏼

**Comment by zprood** [comment]
@bethcollins92 @anthonydillon Thank you. Sorry for missing the URL and just update the last button link address.

**Review by anthonydillon — APPROVED** [review]
Sorry for the delay. LGTM :+1:

**Comment by carkod** [comment]
I just merged some PRs and deployed to prod @bethcollins92  can you rebase? Thanks

**Pull request merged**